### ISSUE TITLE
[JS] [Accessibility] [UserExperience] Provide a way to disable the download option on the Media element

### ISSUE THREAD

**Issue by muthurathinam** [microsoft/AdaptiveCards#5687]
# Platform

JavaScript

# Author or host

Outlook Host

# Version of SDK

Npm [v2.9.0](https://www.npmjs.com/package/adaptivecards/v/2.9.0)

# Details

For a Media element, when audio or video is played the default media player native to the browser/webview is used which comes with its own default set of options. In which download is one such option and this has User Experience(Unable to download the video, UI looks are not aligning with host app i.e outlook and unable to customize them as well) and Accessibility (Focus has not set on download option which is a first option in the flyout for selecting three dots option in media player) issues. Looking for some hooks in the library to disable the download option in the player

**Comment by paulcam206** [comment]
Fixing this issue is not currently on our roadmap. Feel free to continue the conversation on this issue. If you feel strongly about this issue, please let us know and we can reevaluate.
